http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -400,11 +471,157 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       loadBalancerIp <- F.fromOption(
         loadBalancerIpOpt,
         ClusterCreationException(
-          s"Load balancer IP did not become available after ${config.monitorConfig.createIngress.totalDuration} | trace id: ${ctx.traceId}"
+          s"Load balancer IP did not become available after ${config.monitorConfig.createIngress.totalDuration} in cluster ${dbCluster.getGkeClusterId.toString} | trace id: ${ctx.traceId}"
         )
       )
+
+      _ <- logger.info(
+        s"Successfully obtained public IP ${loadBalancerIp.asString} for cluster ${dbCluster.getGkeClusterId.toString} | trace id: ${ctx.traceId}"
+      )
     } yield loadBalancerIp
 
+  private[util] def installGalaxy(appName: AppName,
+                                  dbCluster: KubernetesCluster,
+                                  googleCluster: Cluster,
+                                  nodepoolName: NodepoolName,
+                                  namespaceName: NamespaceName,
+                                  userEmail: WorkbenchEmail,
+                                  customEnvironmentVariables: Map[String, String],
+                                  kubernetesServiceAccount: KubernetesServiceAccount)(
+    implicit ev: ApplicativeAsk[F, AppContext]
+  ): F[Unit] =
+    for {
+      ctx <- ev.ask
+
+      _ <- logger.info(
+        s"Installing helm chart ${config.galaxyAppConfig.chart.value} for app ${appName.value} in cluster ${dbCluster.getGkeClusterId.toString} | trace id: ${ctx.traceId}"
+      )
+
+      helmAuthContext <- getHelmAuthContext(googleCluster, dbCluster.id, namespaceName)
+
+      releaseName = ReleaseName(s"${appName.value}-${config.galaxyAppConfig.releaseNameSuffix.value}")
```

Comment:
would it make sense to suffix a timestamp in the release name?

Reply:
I think it would - especially as we currently have the `keepHistory` option when deleting charts.

Reply:
Hm.. we need `releaseName` at both install and delete time, so if we add a timestamp then we should probably persist it to the database. This may be a good idea anyway, but I just added a TODO for now.